docs/vector_store.ipynb

```diff
@@ -463,6 +463,13 @@
     "print(docs)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note:** You can use the `query` parameter in the Document Loader to apply filters directly when loading documents. This allows you to refine your data selection before performing similarity searches or searching documents without embeddings."
```

Comment:
This might be confusing to a user following the vector store notebook. How is this note to use a document loader with a custom sql query related to the cell for similarity search above?

Reply:
This note may be better under section: "Search for documents with metadata filter", but may need to provide more context to the user about the Document loader. 

Overall, I think we should provide another section "Search for documents without vector search". "You may want to search documents based on Document metadata as a tool or as a part of an exploratory workflow. The Document Loader (add link to colab) can be used to customize the search and load data in the form of Documents from your database. Learn how to 'Load Documents using a SQL query'(add link to colab)". <Add code snippet using page content column  set appropriately as the Document's page content>